src/components/stepper/stepper.jsx

```diff
@@ -15,7 +15,8 @@ module.exports = React.createClass({
     incrementDisplayString: React.PropTypes.string,
     decrementDisplayString: React.PropTypes.string,
     id: React.PropTypes.string,
-    label: React.PropTypes.string
+    label: React.PropTypes.string,
+    readOnly: React.PropTypes.bool
```

Comment:
this new prop is not in the docs - I will add an issue on this repo, maybe we can check this or make the documentation automated soon but for now this one needs to appear in the docs too.